pycbc/inference/option_utils.py

```diff
@@ -445,7 +445,7 @@ def results_from_cli(opts, load_samples=True, walkers=None):
     if load_samples:
         logging.info("Loading samples")
         # check if need extra parameters for a non-sampling parameter
-        file_parameters, cs = transforms.get_conversions(
+        file_parameters, cs = transforms.get_standard_transforms(
```

Comment:
Okay, I see what you call 'standard' now. That's still not what these does though. It returns 'standard inverse' as well. I'd still suggest ``get_transforms``.